src/main/java/fi/vm/yti/terminology/api/v2/mapper/ConceptCollectionMapper.java

```diff
@@ -0,0 +1,138 @@
+package fi.vm.yti.terminology.api.v2.mapper;
+
+import fi.vm.yti.common.dto.ResourceCommonInfoDTO;
+import fi.vm.yti.common.util.MapperUtils;
+import fi.vm.yti.common.util.ModelWrapper;
+import fi.vm.yti.security.YtiUser;
+import fi.vm.yti.terminology.api.v2.dto.ConceptCollectionDTO;
+import fi.vm.yti.terminology.api.v2.dto.ConceptCollectionInfoDTO;
+import org.apache.jena.rdf.model.*;
+import org.apache.jena.riot.RDFDataMgr;
+import org.apache.jena.riot.RDFLanguages;
+import org.apache.jena.vocabulary.*;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.function.Consumer;
+
+public class ConceptCollectionMapper {
+
+    private ConceptCollectionMapper() {
+    }
+
+    public static void dtoToModel(
+            ModelWrapper model,
+            ConceptCollectionDTO dto,
+            YtiUser user) {
+        var modelResource = model.getModelResource();
+        var conceptCollectionResource = model.createResourceWithId(dto.getIdentifier())
+                .addProperty(SKOS.inScheme,
+                        ResourceFactory.createResource(modelResource.getURI()))
```

Comment:
Calling `ResourceFactory.createResource` is redundant in this case, because modelResource is already Resource. 
=> 
`.addProperty(SKOS.inScheme, modelResource)`

Reply:
fixed